core/arch/arm/kernel/link_dummies_init.c

```diff
@@ -6,7 +6,7 @@
 #include <kernel/boot.h>
 #include <mm/core_mmu.h>
 
-unsigned long __section(".text.dummy.get_aslr_seed")
+unsigned long __section(".text.dummy.get_aslr_seed") __weak
```

Comment:
Functions in this file are used to override their normal counterpart as a way of breaking the dependency chain when partitioning the binary for paging. You need to find another way of overriding the `get_aslr_seed()` function.